File: src/Task1.java
import java.util.Scanner;

public class Task1 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Введіть радіус кулі: ");
        float radius = scanner.nextFloat();

        if (radius < 0) {
            System.out.println("Некоректний ввід даних. Радіус не може бути від'ємним.");
        } else {
            // Обчислення об'єму
            float volume = (4f / 3f) * (float) Math.PI * radius * radius * radius;

            // Обчислення площі поверхні
            float surfaceArea = 4f * (float) Math.PI * radius * radius;

            System.out.println("Об'єм кулі: " + volume);
            System.out.println("Площа поверхні кулі: " + surfaceArea);
        }

        scanner.close();
    }
}

File: src/Task_1.kt
import java.util.Scanner
import kotlin.math.PI

fun main() {
    val scanner = Scanner(System.`in`)

    print("Введіть радіус кулі: ")
    val radius = scanner.nextFloat()

    if (radius < 0) {
        println("Некоректний ввід даних. Радіус не може бути від'ємним.")
    } else {
        // Обчислення об'єму
        val volume = (4f / 3f) * PI.toFloat() * radius * radius * radius

        // Обчислення площі поверхні
        val surfaceArea = 4f * PI.toFloat() * radius * radius

        println("Об'єм кулі: $volume")
        println("Площа поверхні кулі: $surfaceArea")
    }

    scanner.close()
}
